sky/backends/backend_utils.py

```diff
@@ -821,6 +823,9 @@ def write_cluster_config(
         with open(yaml_path, 'w') as f:
```

Comment:
Should be `tmp_yaml_path`. Same as L818.

Reply:
Thanks for catching this! fixed